sites/mosul/configs/openmrs/initializer_config/ampathforms/F29-MHPSS_Baseline_v2.json

```diff
@@ -568,7 +568,7 @@
                 "workspaceName": "add-drug-order"
               },
               "hide": {
-                "hideWhenExpression": "wasThePatientPrescribedPsychotropicMedication !== '53eaf1b5-6ff7-498f-b95c-7ce3f116f181'"
+                "hideWhenExpression": "wasThePatientPrescribedPsychotropicMedication !== '53eaf1b5-6ff7-498f-b95c-7ce3f116f181' && wasThePatientPrescribedPsychotropicMedication !== '2d5122f8-24bb-4bce-ab18-9c1e9d224b3c'"
```

Comment:
This could be reversed to specify only the one option that shouldn't be covered

Reply:
Done, please review